wormhole-connect/src/utils/localStorage.ts

```diff
@@ -0,0 +1,56 @@
+import { TransactionLocal } from 'config/types';
+
+// Adds an object to localStorage
+export const addItemToLocalStorage = (
+  id: string, // Key for the item
+  data: object, // Item data
+  prefix: string, // Prefix to use for the item key
+  max?: number, // Max number of items allowed with the same prefix
+) => {
+  const ls = window.localStorage;
+  const items: Array<string> = [];
+
+  // Get all current items with the prefix
+  for (let i = 0; i < ls.length; i++) {
+    const itemKey = ls.key(i);
+    if (itemKey?.toLowerCase().startsWith(prefix)) {
+      items.push(itemKey);
+    }
+  }
+
+  // Remove the oldest items that puts the length at or over the limit
+  if (max && items.length >= max) {
+    for (let i = 0; i < items.length - max + 1; i++) {
+      ls.removeItem(items[i]);
+    }
+  }
```

Comment:
are you just trying to keep a chronological array in localStorage, and trim it to a max length? why not just use a single key and store an array? you could trim it by using `slice`. that way the order is actually reliable. I don't understand the reason for doing it this way

Reply:
I assumed I could depend on the insertion order. I was thinking maybe a timestamp, but using an array with a single entry sounds better.